images/chaos-dlv/README.md

```diff
@@ -6,7 +6,7 @@ is configured properly, you can use it to debug the process on remote machine.
 To deploy the `dlv` image, you need to compile the chaos-mesh with symbols:
 
 ```bash
-make DEBUGGER=1 image
+make DOCKER_REGISTRY="" DEBUGGER=1 image
```

Comment:
Can we make it configurable?

Reply:
actually I am thinking about removing that DOCKER_REGISTRY, using other tools for managing the prefix of docker image. 

Still in finding some eaxmple and best practices.